City-of-Pori/pori-kada
# PORI-197 person card listing

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Import or create person nodes to your local with different information
3. Also add some locations to office taxonomy and assign some people to those locations
4. Go to /phone-book url and you should see a view that lists the persons in a alphabetic order based on last name.
5. Try to filter the list with the free text search and the listing of the offices. Free text search searches from everywhere but the office field.
